bin/hdfcoinc/pycbc_page_snrchi

```diff
@@ -74,8 +74,8 @@ cb.set_label('Trigger Density')
 pylab.xlabel('Signal-to-Noise Ratio')
 pylab.ylabel('Reduced $\\chi^2$')
 pycbc.results.save_fig_with_metadata(fig, args.output_file, 
-     title="%s :SNR vs Reduced %s Chisq" % (ifo, args.chisq_choice),
-     caption="Distribution of SNR and %s Chisq for single detector triggers: "
+     title="%s :SNR vs Reduced %s &chi;<sup>2</sup>" % (ifo, args.chisq_choice),
+     caption="Distribution of SNR and %s &chi;&sup2; for single detector triggers: "
```

Comment:
I'd prefer "single-detector" triggers over "single detector" triggers, (compound adjectives)

However this was in place before, and isn't important, so I'll approve